test/auction.test.js

```diff
@@ -8,6 +8,7 @@ const Auctions = [{
   key: "jest-" + Math.round((new Date()).getTime()),
   source: ["DEMO"],
   target: ["DEMOEX"],
+  options: ['MULTISTEP'],
```

Comment:
This is strange because this don't break other test.

Reply:
@fanshan: The other "options" are not in superstruct yet. They are still being testeed in shaq.
I was hoping Jerome to add test for PKGV2 and V3 but it goes minimalist :)